reference/imap/functions/imap-gc.xml

```
@@ -8,7 +8,7 @@
  <refsect1 role="description">
   &reftitle.description;
   <methodsynopsis>
-   <type>bool</type><methodname>imap_gc</methodname>
+   <type>true</type><methodname>imap_gc</methodname>
```

Changelog needed for 8.0 about invalid flags values now throw a ValueError instead of warning + false return.